com.unity.render-pipelines.high-definition/Editor/Material/ShaderGraph/TargetData/ExposableProperty.cs

```diff
@@ -0,0 +1,143 @@
+using System;
```

Comment:
Is it worthwhile moving this to ShaderGraph repo, so we can use it in URP as well?

Reply:
probably, I put it in hdrp for now for flexibility, but depending on how URP implements it, it could be moved out later